Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -21,47 +21,33 @@ namespace Microsoft.ApplicationInsights.DependencyCollector
     /// </summary>
     public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListener>
     {
-        /// <summary>
-        /// Add Application Insights Dependency Collector services to this .NET Core application.
-        /// </summary>
-        /// <returns>
-        /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
-        /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
-        {
-            if (configuration == null)
-            {
-                configuration = TelemetryConfiguration.Active;
-            }
-
-            return DiagnosticListener.AllListeners.Subscribe(new DependencyCollectorDiagnosticListener(configuration));
-        }
-
         private readonly ApplicationInsightsUrlFilter applicationInsightsUrlFilter;
         private readonly TelemetryClient client;
+        private readonly bool setComponentCorrelationHttpHeaders;
+        private readonly IEnumerable<string> correlationDomainExclusionList;
```

Comment:
This class is setup up in the background by the DependencyTrackingTelemetryModule class when the user calls Initialize().